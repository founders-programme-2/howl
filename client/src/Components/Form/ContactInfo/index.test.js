import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import ContentInfo from './index';

test('Natural', () => {
  let state = {};

  const props = {
    name: '',
    email: '',
    phone: '',
    textChange: name => event => {
      state = { ...state, [name]: event.target.value };
    },
  };
  const contentInfo = renderer
    .create(
      <BrowserRouter>
        <ContentInfo {...props} />
      </BrowserRouter>
    )
    .toJSON();
  expect(contentInfo).toMatchSnapshot();
});
