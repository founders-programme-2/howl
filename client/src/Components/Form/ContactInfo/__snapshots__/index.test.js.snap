// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Natural 1`] = `
Array [
  <h2>
    Contact Info
  </h2>,
  <p>
    Our team of moderators will need to approve your story before it is published publicly. If we have any questions or concerns about your content, we will get in touch using your contact details. We will not use your details to send you spam mail.
  </p>,
  <p
    className="sc-bdVaJa fKlUTO"
  />,
  <div
    className="MuiFormControl-root MuiFormControl-marginNormal MuiTextField-root ContactInfo-textField-1"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated ContactInfo-label-2"
      data-shrink={false}
      htmlFor="name"
    >
      Name (required)
    </label>
    <div
      className="MuiInputBase-root MuiInput-root MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        className="MuiInputBase-input MuiInput-input"
        disabled={false}
        id="name"
        name="name"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="Enter your name..."
        required={false}
        type="text"
        value=""
      />
    </div>
  </div>,
  <p
    className="sc-bdVaJa fKlUTO"
  />,
  <div
    className="MuiFormControl-root MuiFormControl-marginNormal MuiTextField-root ContactInfo-textField-1"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated ContactInfo-label-2"
      data-shrink={false}
      htmlFor="email"
    >
      Email (required)
    </label>
    <div
      className="MuiInputBase-root MuiInput-root MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        className="MuiInputBase-input MuiInput-input"
        disabled={false}
        id="email"
        name="email"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="Enter your email..."
        required={false}
        type="text"
        value=""
      />
    </div>
  </div>,
  <p
    className="sc-bdVaJa fKlUTO"
  />,
  <div
    className="MuiFormControl-root MuiFormControl-marginNormal MuiTextField-root ContactInfo-textField-1"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated ContactInfo-label-2"
      data-shrink={false}
      htmlFor="phoneNum"
    >
      Phone Number (optional)
    </label>
    <div
      className="MuiInputBase-root MuiInput-root MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        className="MuiInputBase-input MuiInput-input"
        disabled={false}
        id="phoneNum"
        name="phone"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="Enter your phone number. e.g:(+44 20 7946 0009)"
        required={false}
        type="text"
        value=""
      />
    </div>
  </div>,
]
`;
